benchmarks/CMakeLists.txt

```diff
@@ -1,37 +1,49 @@
 cmake_minimum_required(VERSION 2.8.3)
-project(moveit_ros_benchmarks)
+project(moveit_benchmarks)
```

Comment:
This is purely an artifact of development.  I had to change the name to make the new package coexist (in catkin) with the existing package.  This can be reverted.